packages/gestalt/src/RadioButton.css

```diff
@@ -1,75 +1,88 @@
 :root {
+  --border-sm: 6px;
+  --border-md: 8px;
+  --border: 2px;
+  --darkGray: #111;
+  --lightGray: #efefef;
   --size-sm: 16px;
   --size-md: 24px;
+  --solid: solid;
+  --translucidGray: rgba(0, 0, 0, 0.12);
 }
 
-.RadioButton {
+.radioButton {
   composes: block from "./Layout.css";
   composes: borderBox from "./Layout.css";
   composes: circle from "./Borders.css";
   composes: flex from "./Layout.css";
   composes: itemsCenter from "./Layout.css";
   composes: justifyCenter from "./Layout.css";
   composes: relative from "./Layout.css";
-  border: 1px solid #8e8e8e;
+  composes: sizeLg from "./Borders.css";
+  composes: solid from "./Borders.css";
+  composes: borderColorLightGray from "./Borders.css";
 }
 
-.RadioButtonSm {
-  height: var(--size-sm);
-  padding: 2px;
-  width: var(--size-sm);
+.radioButton:hover:not(.focused):not(.borderChecked) {
```

Comment:
@AlbertCarreras Don't we also want to change the border color when you hover + it's checked? If so, can you remove `:not(.borderChecked)`?

Reply:
DONE

Reply:
via onMouseEnter/Leave